src/main.rs

```diff
@@ -37,6 +37,28 @@ fn main() -> anyhow::Result<()> {
             .info(CompileTimeInformation::default())
             .print::<Markdown>();
         std::process::exit(0);
+    } else if args.list {
+        let queries = query::SemverQuery::all_queries();
+        let mut rows = vec![["id", "type"], ["==", "===="]];
```

Comment:
The `description` field in the query files is intended to be that kind of short description, though I'm not that skilled at making it nice :)
https://github.com/obi1kenobi/cargo-semver-check/blob/main/src/queries/auto_trait_impl_removed.ron#L4

Feel free to change the descriptions of existing queries if you'd like.

Reply:
Some aren't so short: https://github.com/obi1kenobi/cargo-semver-check/blob/main/src/queries/struct_repr_transparent_removed.ron

Reply:
Good catch. Would it make sense to make an optional "short description" field that we could include here with `short_description.unwrap_or(long_description)` style logic? Then `repr(transparent)` could get a short description and the actually-short ones would be fine omitting it.

Or would you recommend something else, or just skipping for now and coming back to it later? I have no strong feelings one way or the other.

Reply:
I was wanting to get this in with what we have today and then iterate from there.